packages/ado-extension/task.json

```
@@ -29,61 +29,47 @@
         {
             "name": "hostingMode",
             "type": "pickList",
-            "label": "Hosting Mode",
+            "label": "Hosting mode",
             "required": true,
             "options": {
-                "staticSite": "Static Site (this task runs a localhost webserver for your site)",
-                "dynamicSite": "Dynamic Site (you host your site separately)"
+                "staticSite": "Static site (HTML files in a local directory)",
+                "dynamicSite": "URL (localhost, staging URL, or production URL)"
             },
-            "helpMarkDown": "Note: not required in YAML configuration, only on classic pipelines UI. Your site must be served (hosted) before it can be scanned. In _Static Site_ mode, this task will run a localhost webserver serving your site directory and scan that. In _Dynamic Site_ mode, you must host your site yourself separately and specify a URL to scan. This can be either a localhost server you run in an earlier pipeline step, or a remote URL (for example, a staging environment)."
+            "helpMarkDown": "Note: not required in YAML configuration, only on classic pipelines UI. Your site must be served (hosted) before it can be scanned. In _Static site_ mode, this task will run a localhost webserver serving your site directory and scan that. In _URL_ mode, you must host your site yourself separately and specify a URL to scan. This can be either a localhost server you run in an earlier pipeline step, or a remote URL (for example, a staging environment).\n\nWhen switching between hosting modes in the classic pipeline, remove values for the previous hosting mode to prevent caching."
         },
         {
             "name": "staticSiteDir",
             "type": "string",
-            "label": "Static Site Directory",
+            "label": "Hosting mode: site directory",
             "required": true,
-            "helpMarkDown": "Folder containing website content.",
+            "helpMarkDown": "Directory containing the website content to be scanned.",
             "visibleRule": "hostingMode = staticSite"
         },
         {
             "name": "staticSiteUrlRelativePath",
             "type": "string",
-            "label": "Static Site URL Relative Path",
+            "label": "Hosting mode: URL relative path",
             "required": false,
             "defaultValue": "/",
-            "helpMarkDown": "Relative path to directory used to construct base scan url. e.g. / on Ubuntu and // on Windows.",
+            "helpMarkDown": "Relative path to directory used to construct base scan url. For example, use `/` on Ubuntu and `//` on Windows.",
```

This one is super confusing because "relative path" seems like they should put in the path to get to the html file they want scanned. I'm not sure how to explain it better though. Is there a way to maybe put an example here? Something like "if your site directory is ${site_dir} and you use ${relative_path} here, the base scan url will be ${base_scan_url}" so they know what this does?

---

Agreed! I wonder if the label could be:

```
Hosting mode: Base URL
```

And then we could update the description to:

```suggestion
            "helpMarkDown": "Set the base URL when the site lives at a subpath of the domain, such as `/blog`. Otherwise, the base URL should be `/` on Ubuntu and `//` on Windows.",
```


I took inspiration from Ben Balter's definition of baseurl: https://ben.balter.com/jekyll-style-guide/config/#baseurl

---

We should also remove `// on Windows` as that is no longer applicable. Wondering if we should remove defaultValue altogether since it's not required?